tests/MongoDB.Analyzer.Tests.Common.TestCases/Linq/LinqEnums.cs

```diff
@@ -22,137 +22,137 @@ namespace MongoDB.Analyzer.Tests.Common.TestCases.Linq
         }
 
-        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumUInt32\" : 0 }, { \"EnumUInt32\" : 999 }, { \"EnumUInt32\" : -1 }] } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumUInt32\" : 0 }, { \"EnumUInt32\" : 999 }, { \"EnumUInt32\" : -1 }] } }])")]
         public void Enum_uint32()
         {
             _ = GetMongoQueryable<EnumHolder>()
                 .Where(u => u.EnumUInt32 == EnumUInt32.Value0 || u.EnumUInt32 == EnumUInt32.Value999 || u.EnumUInt32 == EnumUInt32.MaxValue);
         }
 
-        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumUInt64\" : NumberLong(0) }, { \"EnumUInt64\" : NumberLong(999) }, { \"EnumUInt64\" : NumberLong(-1) }] } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumUInt64\" : 0 }, { \"EnumUInt64\" : 999 }, { \"EnumUInt64\" : -1 }] } }])")]
         public void Enum_uint64()
         {
             _ = GetMongoQueryable<EnumHolder>()
                 .Where(u => u.EnumUInt64 == EnumUInt64.Value0 || u.EnumUInt64 == EnumUInt64.Value999 || u.EnumUInt64 == EnumUInt64.MaxValue);
         }
 
-        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumInt64\" : NumberLong(0) }, { \"EnumInt64\" : NumberLong(999) }, { \"EnumInt64\" : NumberLong(\"9223372036854775807\") }] } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"$or\" : [{ \"EnumInt64\" : 0 }, { \"EnumInt64\" : 999 }, { \"EnumInt64\" : 9223372036854775807 }] } }])")]
         public void Enum_int64()
         {
             _ = GetMongoQueryable<EnumHolder>()
                 .Where(u => u.EnumInt64 == EnumInt64.Value0 || u.EnumInt64 == EnumInt64.Value999 || u.EnumInt64 == EnumInt64.MaxValue);
         }
 
-        [MQL("aggregate([{ \"$match\" : { \"$or\" : [{ \"Vehicle.VehicleType.Type\" : 0 }, { \"Vehicle.VehicleType.Type\" : 2 }] } }, { \"$project\" : { \"LicenceNumber\" : \"$Vehicle.LicenceNumber\", \"_id\" : 0 } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"$or\" : [{ \"Vehicle.VehicleType.Type\" : 0 }, { \"Vehicle.VehicleType.Type\" : 2 }] } }, { \"$project\" : { \"_v\" : \"$Vehicle.LicenceNumber\", \"_id\" : 0 } }])")]
         public void Enum_not_typed_in_nested_class()
         {
             _ = GetMongoQueryable<Person>()
-                .Where(u => u.Vehicle.VehicleType.Type == VehicleTypeEnum.Bus || u.Vehicle.VehicleType.Type == VehicleTypeEnum.Motorcylce)
+                .Where(u => u.Vehicle.VehicleType.Type == VehicleTypeEnum.Bus || u.Vehicle.VehicleType.Type == VehicleTypeEnum.Motorcycle)
                 .Select(u => u.Vehicle.LicenceNumber);
         }
 
-        [MQL("aggregate([{ \"$match\" : { \"MPG\" : { \"$gt\" : 20.0 } } }, { \"$project\" : { \"Item1\" : \"$VehicleMake\", \"Item2\" : \"$Type\", \"_id\" : 0 } }])")]
+        [MQL("Aggregate([{ \"$match\" : { \"MPG\" : { \"$gt\" : 20.0 } } }, { \"$project\" : { \"_v\" : [\"$VehicleMake\", \"$Type\"], \"_id\" : 0 } }])")]
```

Comment:
Lots of this _v replacement again. Not sure why this is happening, that would be doc version number which doesn't seem equivalent.